src/adagrams.js

```diff
@@ -1,15 +1,212 @@
+// an array prototype to return a count of how many times an item occurs in an array
+Array.prototype.count = function(value) {
```

Comment:
You will definitely see stack overflow recommendations to extend prototype but there are some compelling reasons not to do so. Feel free to investigate that on your own, if you're curious! The better choice is to simply define a function that accepts an array and an item to count.